src/CommentSold/Api/Clients/Rest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommentSold\Api\Clients;
+
+use CommentSold\Api\Exception\CommentSoldException;
+use CommentSold\Api\Response;
+use GuzzleHttp\Client;
+
+class Rest
+{
+    private const TOKEN_SERVICE_URL = 'https://tokens.cs-api.com';
+    private const API_URL = 'https://openapi.commentsold.com/v1';
+
+    public const PER_PAGE = 10;
```

Comment:
I don't think the default per page should be in this low-level class.

(probably not the endpoints either)

Reply:
So remove the SDK default and let the OpenAPI defaults handle it?

Reply:
You could. Or make it a private constant in ShopClient and GlobalClient. Those classes are higher-level and have some context for the endpoint options.